ipykernel/kernelspec.py

```diff
@@ -82,7 +83,8 @@ def write_kernel_spec(path=None, overrides=None):
     return path
 
 
-def install(kernel_spec_manager=None, user=False, kernel_name=KERNEL_NAME, display_name=None, prefix=None):
+def install(kernel_spec_manager=None, user=False, kernel_name=KERNEL_NAME, display_name=None,
+            profile=None, prefix=None):
```

Comment:
Can you add profile on the end instead of just before prefix, so that it's a backward-compatible patch in the extremely unlikely event of people passing all arguments positionally?